types.ts

```diff
@@ -213,18 +213,17 @@ export interface RelationshipPerson {
   id: number;
   firstName: string;
   lastName: string;
+  gender?: string;
```

Comment:
Why optional?

Reply:
We decided that the gender is optional in the first iteration